perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -169,6 +169,8 @@ sub admin_pages {
         $pages->{defecttype_edit} = [ undef, undef ];
     };
 
+    $pages->{body} = [ undef, undef ];
```

Comment:
This is a bit of a hack to get the `defect_types_for_category` url to show up in `check_page_allowed`